tasks/libs/ciproviders/github_api.py

```diff
@@ -519,14 +519,29 @@ def get_token_from_app(app_id_env='GITHUB_APP_ID', pkey_env='GITHUB_KEY_B64'):
         auth_token = integration.get_access_token(install_id)
         print(auth_token.token)
 
-    def create_label(self, name, color, description=""):
+    def create_label(self, name, color, description="", ignore_existing=False):
```

Comment:
Not sure `ignore_existing` is a great name for this. Let me know if you have a better idea

Reply:
You could take inspiration from the stdlib and use `exist_ok` https://docs.python.org/3/library/os.html#os.makedirs.

Reply:
👍